packages/contracts/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "test": "hardhat test",
     "build": "hardhat compile",
-    "build:npm": "rollup --config rollup.config.ts",
+    "build:npm": "rollup --config rollup.config.ts && cp -a ./src/* ./dist",
```

Comment:
```suggestion
    "build:npm": "rollup --config rollup.config.ts && cp -a ./src/core ./ && cp -a ./src/framework . && cp -a ./src/utils . &&  cp -a ./src/plugins . && cp -a ./src/token .",
```
Wouldn't this work?

Reply:
the package.json and/or `.npmignore` files need to be updated for this

And this script line will have to be updated if we ever add another folder

Reply:
I would do it to keep the import paths as short as possible.
Including `dist` for no reason is annoying/ugly.